docker/etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official

```diff
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Do we need these files, or could they potentially be moved into a "container" directory instead of "docker"?

Reply:
Moved to container directory.

Reply:
ea931db